### DISCUSSION TITLE
閲覧数が表示されなくなりました。

### DISCUSSION THREAD

**Discussion by a-kashiwagi** [increments/qiita-discussions#507] [要望・アイデア・不具合]
数日前に、Qiita のレイアウトが微妙に変更になったようで、
記事の頭にアカウント名が表示されるようになったような
のですが、Web 版の、自アカウントにログイン時に表示さ
れていた「閲覧数」が無くなってしまいました。

どこか別の場所に表示されるようになったのでしょうか？

宜しくお願い致します。

**Comment by tanaka-asami415** [comment]
@a-kashiwagi 
お問い合わせありがとうございます！
下記画像の場所に表示されておりますので、ご確認いただけますと幸いです🙏

![image](https://github.com/increments/qiita-discussions/assets/103916185/9d70560a-1b36-4a65-8ac7-710450a59374)

**Reply by a-kashiwagi** [reply]
# おおっ、こんなところに！ 気づきませんでした。
# 代りに Google analytics を設定してしまいました。

わかりました。ありがとうございます。
結構、探したのですが目に入りませんでした。

宜しくお願い致します。
